RFCs/RFC1.md

```diff
@@ -0,0 +1,38 @@
+# RFC1: Structure and organization of The Carpentries
+
+## Previous motion, approved by the Software Carpentry (SWC) Steering Committee and Data Carpentry (DC) Board:
+
+*The Carpentries is composed of the following basic structure: A Carpentry umbrella organization (hereafter 
+referred to as The Carpentries) and a set of Lesson Carpentries (Lesson Organizations). The umbrella organization 
+is primarily responsible for finances, administration, instructor training and facilitating policy.  Each Lesson 
+Carpentry (Lesson Organization) is primarily responsible for lesson development and maintenance. 
+(Approved [July 3](https://github.com/swcarpentry/board/blob/master/minutes/minutes-2017-07-03.md))*
+
+## Current vision of responsibilities for The Carpentries:
+* Operations:
+  * Coordinating with partner organizations and grant agencies (negotiating contracts, allocation of resources, 
+  	fulfillment of obligations)
+  * Maintaining budget and reporting on finances
+  * Appropriately allocating staff time to meet the needs of The Carpentries community and lesson organizations
+  * Managing duplication and overlap (minimizing competition and mediating conflict between lesson organizations)
```

Comment:
* Instructor membership management and maintaining records of Carpentries instructors